happi/client.py

```diff
@@ -391,9 +391,9 @@ def _get_search_results(self, items, *, wrap_cls=None):
             try:
                 result = wrap_cls(client=self, device=self.find_device(**info))
                 results.append(result)
-            except Exception:
-                logger.warning('Entry for %s is malformed. Skipping.',
-                               info['name'])
+            except Exception as exc:
+                logger.warning('Entry for %s is malformed (%s). Skipping.',
+                               info['name'], str(exc))
```

Comment:
`'%s' % obj` is implicitly `str(obj)` (as opposed to `%r` -> `repr(obj)`)
```suggestion
                               info['name'], exc)
```

Reply:
Thanks for the review!